### PR TITLE
Update test projects to netcoreapp3.1 in master

### PR DESCRIPTION
**Pull request by bording**
For your review @mauroservienti

### CODE REVIEW

File: src/NServiceBus.ContainerTests/NServiceBus.ContainerTests.csproj [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" PrivateAssets="All" />
-    <PackageReference Include="NUnit" Version="[3.12.0, 4.0.0)" />
```
Comment (danielmarbach) [inline src/NServiceBus.ContainerTests/NServiceBus.ContainerTests.csproj]: Aren't container tests shipped as nuget?

File: src/NServiceBus.TransportTests/NServiceBus.TransportTests.csproj [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" PrivateAssets="All" />
-    <PackageReference Include="NUnit" Version="[3.12.0, 4.0.0)" />
```
Comment (danielmarbach) [inline src/NServiceBus.TransportTests/NServiceBus.TransportTests.csproj]: Aren't transport tests shipped as nuget?

Comment (andreasohlund) [reply src/NServiceBus.TransportTests/NServiceBus.TransportTests.csproj]: they are, why would we need a range though?

Comment (danielmarbach) [reply src/NServiceBus.TransportTests/NServiceBus.TransportTests.csproj]: Why did we add it in the first place?

**Comment by bording** [comment]
@danielmarbach The script I used to generate the PRs didn't properly account for test projects that also produce packages, so I'm going to be manually fixing up this PR.

**Pull request merged**